packages/cli/src/lib/snapshot/snapshot.ts

```diff
@@ -68,6 +69,12 @@ export class Snapshot {
     await this.client.resourceSnapshot.delete(this.model.id);
   }
 
+  public async download() {
```

Comment:
`async` is not needed here since there's no `await` keyword in the function body